### PR TITLE
drop obsolete code with .tar.gz archive

### PR DESCRIPTION
**Pull request by agalitsyn**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -20,17 +20,12 @@ IMPORT_OPTIONS := --vendor \
 		--repository=$(REPOSITORY) \
 		--name=$(NAME) \
 		--version=$(VER) \
-		--glob=**/*.yaml \
-		--ignore=dev \
+		--glob=resources/*.yaml \
```
Comment (sofuture) [inline Makefile]: I think this might need to be `**/*.yaml` still, and `--ignore=dev` is still needed as well.

Comment (sofuture) [inline Makefile]: No, I take that back.

**Comment by sofuture** [comment]
LGTM

File: Makefile (hunk 1 above)
Comment (agalitsyn) [inline Makefile]: It's weird that after this change  import is broken
```
$ gravity app import --vendor --ops-url=https://opscenter.localhost.localdomain:33009 --insecure --repository=gravitational.io --name=stolon-app --version=0.0.5 --glob=resources/*.yaml --registry-url=apiserver:5000 --set-image=stolon-bootstrap:0.0.5 --set-image=stolon-uninstall:0.0.5 --set-image=stolon:0.0.5 --set-image=stolon-hatest:0.0.5 --set-dep=gravitational.io/k8s-onprem:$(gravity app list --ops-url=https://opscenter.localhost.localdomain:33009 --insecure | grep -m 1 k8s-onprem | awk '{print $3}' | cut -d: -f2 | cut -d, -f1) --debug .
INFO[0000] evaluating default for state directory, determine operation mode  file=gravity/main.go:863 func=main.newLocalApp
INFO[0000] operating in installed mode, setting state dir as /var/lib/gravity/local  file=gravity/main.go:874 func=main.newLocalApp
DEBU[0000] got search paths: [/var/lib/gravity assets/local]  file=process/cfg.go:31 func=process.ReadConfig
DEBU[0000] look up configs in /var/lib/gravity           file=process/cfg.go:34 func=process.ReadConfig
DEBU[0000] /var/lib/gravity/gravity.yaml not found in search path  file=process/cfg.go:31 func=process.ReadConfig
DEBU[0000] look up configs in assets/local               file=process/cfg.go:34 func=process.ReadConfig
DEBU[0000] assets/local/gravity.yaml not found in search path  file=process/cfg.go:31 func=process.ReadConfig
WARN[0000] creating insecure client                      file=gravity/main.go:770 func=main.(*localEnvironment).getClient
[==========>                                                                                         ] 10% (vendoring docker images)      INFO[0000] found resource files: []                      file=service/vendor.go:78 func=service.(*vendorer).Vendor
INFO[0000] registry layers are missing, pulling & exporting the following images: []  file=service/vendor.go:96 func=service.(*vendorer).Vendor
ERRO[0004] 
ERROR REPORT:
Original Error: *trace.NotFoundError package(gravitational.io/k8s-onprem:0.0.1) not found
Stack Trace:
	/home/agalitsyn/Projects/go/src/github.com/gravitational/gravity/lib/storage/keyval/repositories.go:109 github.com/gravitational/gravity/lib/storage/keyval.(*backend).GetPackage
	/home/agalitsyn/Projects/go/src/github.com/gravitational/gravity/lib/storage/keyval/applications.go:10 github.com/gravitational/gravity/lib/storage/keyval.(*backend).GetApplication
	/home/agalitsyn/Projects/go/src/github.com/gravitational/gravity/lib/app/service/app.go:276 github.com/gravitational/gravity/lib/app/service.(*applications).GetApp
	/home/agalitsyn/Projects/go/src/github.com/gravitational/gravity/lib/app/service/app.go:505 github.com/gravitational/gravity/lib/app/service.(*applications).resolveManifest
	/home/agalitsyn/Projects/go/src/github.com/gravitational/gravity/lib/app/service/app.go:362 github.com/gravitational/gravity/lib/app/service.(*applications).CreateImportOperation
	/home/agalitsyn/Projects/go/src/github.com/gravitational/gravity/lib/app/acl.go:34 github.com/gravitational/gravity/lib/app.(*ApplicationsACL).CreateImportOperation
	/home/agalitsyn/Projects/go/src/github.com/gravitational/gravity/lib/app/handler/handler.go:142 github.com/gravitational/gravity/lib/app/handler.(*WebHandler).createImportOperation
	/home/agalitsyn/Projects/go/src/github.com/gravitational/gravity/lib/app/handler/handler.go:54 github.com/gravitational/gravity/lib/app/handler.(*WebHandler).(github.com/gravitational/gravity/lib/app/handler.createImportOperation)-fm
	/home/agalitsyn/Projects/go/src/github.com/gravitational/gravity/lib/app/handler/handler.go:692 github.com/gravitational/gravity/lib/app/handler.(*WebHandler).needsAuth.func1
	/home/agalitsyn/Projects/go/src/github.com/gravitational/gravity/vendor/github.com/julienschmidt/httprouter/router.go:300 github.com/gravitational/gravity/vendor/github.com/julienschmidt/httprouter.(*Router).ServeHTTP
	/home/agalitsyn/Projects/go/src/github.com/gravitational/gravity/vendor/github.com/julienschmidt/httprouter/router.go:238 github.com/gravitational/gravity/vendor/github.com/julienschmidt/httprouter.(*Router).Handler.func1
	/home/agalitsyn/Projects/go/src/github.com/gravitational/gravity/vendor/github.com/julienschmidt/httprouter/router.go:300 github.com/gravitational/gravity/vendor/github.com/julienschmidt/httprouter.(*Router).ServeHTTP
	/usr/local/go/src/net/http/server.go:1863 net/http.serverHandler.ServeHTTP
	/usr/local/go/src/net/http/server.go:1362 net/http.(*conn).serve
	/usr/local/go/src/runtime/asm_amd64.s:1722 runtime.goexit
User Message: package(gravitational.io/k8s-onprem:0.0.1) not found
  file=gravity/main.go:920 func=main.printError
[ERROR]: package(gravitational.io/k8s-onprem:0.0.1) not found
```

So I will return it back in another PR

**Pull request merged**